### PR TITLE
Webui review

### PR DESCRIPTION
**Pull request by adrianschroeter**
some minor  bits during reviewing Package/Project find method usages in webui

### CODE REVIEW

**Comment by ChrisBr** [comment]
LGTM

File: src/api/app/controllers/webui/request_controller.rb [hunk 1]
```diff
@@ -169,10 +169,14 @@ def changerequest
           if tpkg
             target = Package.find_by_project_and_name(tprj, tpkg)
           else
-            target = Project.Project.find_by_name tprj
```
Comment (bgeuken) [inline src/api/app/controllers/webui/request_controller.rb]: How did this get into our codebase? :astonished:

Comment (bgeuken) [inline src/api/app/controllers/webui/request_controller.rb]: Oh, it's from [2013](374d6bea534d4fe7ee731389d2169b0ae10fcb7a). Is this codepath used at all? It's certainly misses a test...

Comment (adrianschroeter) [inline src/api/app/controllers/webui/request_controller.rb]: On Mittwoch, 21. Oktober 2015, 02:27:33 CEST wrote Bj�rn Geuken:
> > @@ -169,10 +169,14 @@ def changerequest
> >            if tpkg
> >              target = Package.find_by_project_and_name(tprj, tpkg)
> >            else
> > -            target = Project.Project.find_by_name tprj
> 
> Oh, it's from [2013](374d6bea534d4fe7ee731389d2169b0ae10fcb7a). Is this codepath used at all? It's certainly misses a test...

I am not aware of a real life use case for this, afaik it is only used
for packages so far.

However, we either need to check for the request action kinds or deal with this
situation somehow else.

But I would leave this for a generic webui request handling refactoring. The
current code is mostly anyway broken when having multiple actions.

**Pull request merged**